stencils/Subject.stencil

```diff
@@ -8,7 +8,7 @@ typealias BoolSubject = MappedSubject<KotlinBoolean, Bool, BoolMapper>
 typealias IntSubject = MappedSubject<KotlinInt, Int, IntMapper>
 typealias FloatSubject = MappedSubject<KotlinFloat, Float, FloatMapper>
 typealias DoubleSubject = MappedSubject<KotlinDouble, Double, DoubleMapper>
-typealias DateSubject = MappedSubject<KalugaDate, Foundation.Date, DateMapper>
+// typealias DateSubject = MappedSubject<KalugaDate, Foundation.Date, DateMapper>
```

Comment:
Why it's commented? I will break for sure existing code which uses this modules.
Maybe you have to use DefaultKalugaDate here?

Reply:
The commended out code is so the generated code could compile for an xBike release but will not be the final product. I did not try DefaultKalugaDate as it made no sense to me.